src/App.tsx

```diff
@@ -138,6 +139,10 @@ const AppRoutes = ({ base }: AppProps) => {
                     </Route>
                   </Route>
                 </Route>
+                <Route
+                  path="resources/images/:imageId"
+                  element={<ImagePage />}
+                />
```

Comment:
Generell kommentar: Når alle disse blir merget hadde det vært greit om du laget en nested route for alle `/resources/`-endepunktene her.